Command/RemoveThumbsCommand.php

```diff
@@ -141,33 +134,29 @@ public function execute(InputInterface $input, OutputInterface $output)
         $this->log("Done (total medias processed: {$totalMediasCount}).");
     }
 
-    /**
-     * @return MediaProviderInterface
-     */
-    public function getProvider()
+    public function getProvider(): MediaProviderInterface
```

Comment:
Adding return types (on a non-final class) is a BC break!

https://3v4l.org/aN79d

Reply:
recover the old signature please for all the methods except the one you created

Reply:
Oh sorry @vyshkant that's my fault